accelerator/cli_info.h

```diff
@@ -69,6 +71,7 @@ static struct ta_cli_argument_s {
     {"iri_port", required_argument, NULL, IRI_PORT_CLI, "IRI listening port"},
     {"mqtt_host", required_argument, NULL, MQTT_HOST_CLI, "MQTT listening host"},
     {"mqtt_root", required_argument, NULL, MQTT_ROOT_CLI, "MQTT listening topic root"},
+    {"iri_url_list", required_argument, NULL, IRI_URL_LIST_CLI, " List of IRI listening URL"},
```

Comment:
`iri_url_list` sounds weird. Is it equivalent to `iri_host_list` or `list_iri_host`?

Reply:
It is the URL concatenate host and port. The command would be in this style `--iri_url_list localhost:8000`

Reply:
IP address and port pair is called, [Socket Address](https://en.wikipedia.org/wiki/Network_socket).
Don't use URL because it does not match what you did.

Reply:
Cc. @marktwtn for clarifying the term "socket address."

Reply:
So should I name this command as `iri_socket_address`?